src/main/java/com/coveros/selenified/Selenified.java

```diff
@@ -519,6 +521,47 @@ private void setupScreenSize(App app) {
         }
     }
 
+
+    /**
+     * looks at the browser information passed in, and loads that data into a
+     * list
+     *
+     * @return List: a list of all browser
+     * @throws InvalidBrowserException If a browser that is not one specified in the
+     *                                 Selenium.Browser class is used, this exception will be thrown
+     */
+    private static List<Browser> getBrowserInput() throws InvalidBrowserException {
+        List<Browser> browsers = new ArrayList<>();
+        // null input check
+        if (System.getProperty(BROWSER_INPUT) == null) {
+            return browsers;
+        }
+        String[] browserInput = System.getProperty(BROWSER_INPUT).split(",");
+        for (String singleBrowserInput : browserInput) {
+            browsers.add(new Browser(singleBrowserInput));
+        }
+        return browsers;
+    }
+
+    /**
+     * Obtains passed in browser information, and sets up the required
+     * capabilities
+     *
+     * @throws InvalidBrowserException If a browser that is not one specified in the
+     *                                 Selenium.Browser class is used, this exception will be
+     *                                 thrown
+     */
+    private static void setupTestParameters() throws InvalidBrowserException {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this method into "MasterSuiteSetupConfigurator". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS3398)